HostsPrefpane/HostsPrefpane-Prefix.pch

```diff
@@ -5,3 +5,8 @@
 #ifdef __OBJC__
     #import <Cocoa/Cocoa.h>
 #endif
+
+// In case we run the Hosts code as a Pref Pane, NSLocalizedString does not help
+#define NSLocalizedStringUniversal(key, comment) \
+[[NSBundle bundleForClass: [self class]] localizedStringForKey:(key) value:@"" table:nil]
```

Comment:
You would think there should be a different between prefpane and app

Reply:
Yes. I will remove that and use NSBundle bundleForClass directly.

Reply:
Removed NSLocalizedStringUniversal